azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_azurerm.go

```diff
@@ -146,12 +149,18 @@ func ResourceServiceEndpointAzureRM() *schema.Resource {
 			Version: 1,
 		},
 	}
+
+	r.Schema["validate"] = &schema.Schema{
```

Comment:
Can we wrapper `validate` into a `features` block like [build definition features](
https://github.com/microsoft/terraform-provider-azuredevops/blob/main/azuredevops/internal/service/build/resource_build_definition.go#L346) ? This property is a feature not part of the API.